src/atom.ts

```diff
@@ -7,6 +7,41 @@ import equal from './equal'
 import cachedSubject from './cached-subject'
 import take from 'callbag-take'
 
+type DevToolsType = {
+  connect: (options: {
+    name: string
+  }) => {
+    subscribe: () => () => {}
+    send: (action: unknown, state: unknown) => void
+  }
+}
+const devTools = ((window as any)
+  .__REDUX_DEVTOOLS_EXTENSION__ as unknown) as DevToolsType
+
+const devToolsInstance = devTools.connect({ name: 'David suxx' })
+
+const subscribedValues: Record<string, unknown> = {}
+
+const initializeAtom = (atomName: string, newValue: unknown) => {
+  if (Object.keys(subscribedValues).includes(atomName)) {
+    return
+  }
+  subscribedValues[atomName] = newValue
+  devToolsInstance.send({ type: `INITIALIZE ${atomName}` }, subscribedValues)
+}
+
+const updateValue = (atomName: string, newValue: unknown) => {
+  subscribedValues[atomName] = newValue
+  devToolsInstance.send({ type: `UPDATE ${atomName}` }, subscribedValues)
+}
+
+const removeValue = (atomName: string) => {
+  delete subscribedValues[atomName]
+  devToolsInstance.send({ type: `REMOVE ${atomName}` }, subscribedValues)
+}
```

Comment:
To differentiate more from the actual atom CRUD:ing, how about these alternative names:

* `initializeAtom` -> `registerAtomCreation`
* `updateValue` -> `registerAtomUpdate`
* `removeValue` -> `registerAtomCleanup`